src/codecoverage/bot/requirements_frozen.txt

```diff
@@ -48,10 +59,12 @@ python-dateutil==2.6.1
 python-hglib==2.6.1
 pytoml==0.1.20
 pytz==2018.9
+pytz==2019.1
```

Comment:
This and urllib3 is present twice in this file

Reply:
This is what happens when I have to run pypi2nix manually, and import the result...